external/netutils/Kconfig.netutil

```diff
@@ -8,3 +8,12 @@ config NETUTILS_NETLIB
 	default n
 	---help---
 		Enable support for the network support library.
+
+if NETUTILS_NETLIB
+config NET_NETMON
+	bool "Enable network monitor"
+	default y
+	depends on NET_LWIP
```

Comment:
I think netmon should not be dependent  to NET_LWIP

Reply:
I agree with you. Modification completed.